<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DlgExclude</class>
 <widget class="QDialog" name="DlgExclude">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>500</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>500</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Excluded items</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normalon>:/img/UI/rsc/exclude.png</normalon>
   </iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">font: 10pt &quot;Segoe UI Symbol&quot;;</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <property name="leftMargin">
    <number>9</number>
   </property>
   <property name="topMargin">
    <number>9</number>
   </property>
   <property name="rightMargin">
    <number>9</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Excluded files</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="10,1">
      <item>
       <widget class="QListWidget" name="excludedFilesControl"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QPushButton" name="btnBrowseFile">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>Browse...   </string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/img/UI/rsc/browse.png</normalon>
           </iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnRemoveFile">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>Remove       </string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/img/UI/rsc/cancel.png</normalon>
           </iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtFile">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnFileWild">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>Add wildcard</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/img/UI/rsc/wildcard.png</normalon>
           </iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_2">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>0</height>
      </size>
     </property>
     <property name="title">
      <string>Exclude files having any of thes attribues set:</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_4">
      <item>
       <widget class="QCheckBox" name="chkH">
        <property name="text">
         <string>Hidden</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkS">
        <property name="text">
         <string>System</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QCheckBox" name="chkE">
        <property name="text">
         <string>Encrypted</string>
        </property>
       </widget>
      </item>
      <item>
       <spacer name="horizontalSpacer_2">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>232</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="groupBox_3">
     <property name="title">
      <string>Excluded folders</string>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="10,1">
      <item>
       <widget class="QListWidget" name="excludedFoldersControl"/>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QPushButton" name="btnBrowseFolder">
          <property name="minimumSize">
           <size>
            <width>90</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>Browse...   </string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/img/UI/rsc/browse.png</normalon>
           </iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnRemoveFolder">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>Remove      </string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/img/UI/rsc/cancel.png</normalon>
           </iconset>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="txtFolder">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>28</height>
           </size>
          </property>
          <property name="maximumSize">
           <size>
            <width>16777215</width>
            <height>16777215</height>
           </size>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="btnFolderWild">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>32</height>
           </size>
          </property>
          <property name="text">
           <string>Add wildcard</string>
          </property>
          <property name="icon">
           <iconset>
            <normalon>:/img/UI/rsc/wildcard.png</normalon>
           </iconset>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="btnOK">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>OK</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="btnCancel">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>32</height>
        </size>
       </property>
       <property name="text">
        <string>Cancel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../rsc.qrc"/>
 </resources>
 <connections/>
</ui>
